eventiq_asb/service.py

```diff
@@ -0,0 +1,15 @@
+from typing import ClassVar
+
+from eventiq import Service as EventiqService
+from eventiq.types import MiddlewareType
+
+from .middlewares import ReceiverMiddleware
+
+
+class Service(EventiqService):
```

Comment:
Service is "final" and shouldn't be extended like this. Modify `Service.default_middlewares` directyl for example in `__init__`

Reply:
removed service.py, moved this part on broker